app/models/comment.rb

```diff
@@ -0,0 +1,15 @@
+class Comment < ActiveRecord::Base
+
+  include ActsAsCommentable::Comment
+  include ApplicationHelper
+
+  belongs_to :commentable, :polymorphic => true
+
+  default_scope -> { order('created_at ASC') }
+
+  belongs_to :user
+
+  def human_date
```

Comment:
I'd make this a view helper rather than "pollute" the model with display concerns.